grails-app/services/org/pih/warehouse/fulfillment/FulfillmentService.groovy

```diff
@@ -378,4 +379,71 @@ class FulfillmentService {
 
         return debitTransaction
     }
+
+    String bindOrInferLotNumber(ImportPackingListItem obj, Map source) {
```

Comment:
I can see that the only property from `obj` we use is the `origin`. Let's change the arguments to be `Location origin, Map source`, since we do not need anything else from the `obj`.

Reply:
let's leave it as they are since the methods are basically the `@UseBinding` methods but moved to the service.
The other binding method requires this `obj` since we set `obj.binLcoationNotFound` flag on it and it would be weird where one has this argument and the other binding method doens't